lib/deck.rb

```diff
@@ -0,0 +1,28 @@
+class Deck
+  attr_reader :cards
+  def initialize(cards=[])
+  @cards = cards
+  end
+
+  def rank_of_card_at(index)
+    @cards[index].rank
+  end
+
+  def high_ranking_cards
+    @cards.map do |card|
+         card if card.rank >= 11   
+    end.compact
+  end
+  
+  def percent_high_ranking
+    ( ( high_ranking_cards.count * 100 ).to_f / @cards.count.to_f ).round(2)
```

Comment:
With Ruby math, either the numerator OR denominator needs to be a float, but not both. :) So you can take out one of your `to_f` s

Reply:
Oh yeah, that makes sense! thank you